models/transaction.py

```diff
@@ -0,0 +1,464 @@
+import json
+from datetime import datetime, date
+from typing import Literal, Optional
+from utils import date_utils
+from models import *
+
+
+class OriginatedAchTransactionDTO(object):
```

Comment:
create a base class transactionDTO and let all transactions to inherit from.
base fields:
id
createdAt
account (relationship)
customer (relationship)
customers (relationship)
org (relationship)
amount
direction
balance
summary
tags